drivers/serial/uart_gecko.c

```diff
@@ -380,35 +380,36 @@ static void usart_gecko_config_func_0(struct device *dev);
 #endif
 
 static const struct uart_gecko_config usart_gecko_0_config = {
-	.base = (USART_TypeDef *)DT_USART_GECKO_0_BASE_ADDRESS,
+	.base = (USART_TypeDef *)DT_SILABS_GECKO_USART_USART_0_BASE_ADDRESS,
```

Comment:
The `USART` string repetition in the generated #define seems unnecessary to me.
@galak, is there a way to avoid this?

Reply:
This pattern was established by #11180. The first part 'SILABS_GECKO_USART' is equal to the 'compatible' property of the node for which the define is generated. What follows is the extracted property itself.

One could argue that this is not the right choice for the define generated from an alias node since according to the device tree specification "The alias node shall be at the root of the devicetree". That means it belongs to the global namespace.

Reply:
@mnkp,

I definitely agree on using aliases over dts_fixup. It is just strange for me (almost hurt my eyes) seeing this string repetitions.